apps/form.go

```diff
@@ -1,11 +1,39 @@
 package apps
 
+// Form defines what inputs a Call accepts, and how they can be gathered from
```

Comment:
"Form defines what inputs a Call accepts" - this can be hard to grok for a developer (considering that a Form also contains a Call).